101-pubnub-eventhub-bridge/README.md

```diff
@@ -0,0 +1,245 @@
+#  PubNub Realtime Gateway for Azure Event Hubs
+
+## What does it do?
+The PubNub Realtime Gateway for Azure provides a realtime data stream bridge between the PubNub Data Stream Network and Azure Event Hubs. -- consider it a bi-directional bridge between PubNub and Azure!
+
+If you need to:
+
+* Feed realtime PubNub data into Azure via an Event Hub, and then use that Event Hub as an input for other Azure cloud services
+* Feed Azure Event Hub data back out to PubNub
+
+Then this is the ARM template for you!
+
+Tags: ``Event Hubs, Event Hub, Realtime, PubNub, PubSub, Pub/Sub, Publish/Subscribe, Node.js, Web Jobs``
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fazuredeploy.json" target="_blank">
```

Comment:
This link needs to point at the location the template will be at; for instance, for the 201-vmss-linux-nat template, the url is this: https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-vmss-linux-nat%2Fazuredeploy.json